diff --git a/dist/experimental/testmode/fetch.js b/dist/experimental/testmode/fetch.js
index 71b25d5fb743920f0d41767bf82b0b26d58ecd98..5a5e9b9edf951c723d6cbcbbac68e073a2beab9d 100644
--- a/dist/experimental/testmode/fetch.js
+++ b/dist/experimental/testmode/fetch.js
@@ -65,7 +65,7 @@ async function buildProxyRequest(testData, request) {
                     getTestStack()
                 ]
             ],
-            body: body ? Buffer.from(await request.arrayBuffer()).toString('base64') : null,
+            body: body ? Buffer.from(await request.clone().arrayBuffer()).toString('base64') : null,
             cache,
             credentials,
             integrity,
diff --git a/image-types/global.d.ts b/image-types/global.d.ts
index 661c316e89b3c1aa5913b8afebdbf3e6299eccd7..290d6b3774ac9cb71bf6ba5f458e8054de901669 100644
--- a/image-types/global.d.ts
+++ b/image-types/global.d.ts
@@ -13,7 +13,7 @@ declare module '*.svg' {
    * `@svgr/webpack` plugin or
    * `babel-plugin-inline-react-svg` plugin.
    */
-  const content: any
+  const content: import('../dist/shared/lib/image-external').StaticImageData
 
   export default content
 }
